config.py.sample

```diff
@@ -37,6 +37,9 @@ MUSICBRAINZ_CLIENT_SECRET = "CLIENT_SECRET"
 LASTFM_API_KEY = "USE_LASTFM_API_KEY"
 
 
+# Max time in seconds after which the playing_now stream will expire.
+PLAYING_NOW_MAX_DURATION = 5 * 60
```

Comment:
the playing_now song is stored with userid as the key, so it gets overwritten when the user listens to a new song with the new expiration as set by us. This is already implemented and works fine.
I'll change the value to 10 mins.